restapi/src/main/java/io/stargate/web/docsapi/resources/ReactiveDocumentResourceV2.java

```diff
@@ -319,6 +358,11 @@ public void putDocPath(
           @NotNull(message = "payload not provided")
           @NotBlank(message = "payload must not be empty")
           String payload,
+      @ApiParam(
+              value = "Include this to make the TTL match that of the parent document.",
```

Comment:
can we add that this requires read before write if `true`.. Also to the other methods with `ttlAuto`